app/src/main/java/io/gnosis/safe/di/components/ApplicationComponent.kt

```diff
@@ -19,18 +19,19 @@ import io.gnosis.safe.ui.base.fragment.BaseDialogFragment
 import io.gnosis.safe.ui.base.fragment.BaseFragment
 import io.gnosis.safe.ui.terms.TermsChecker
 import io.gnosis.safe.utils.BalanceFormatter
+import io.gnosis.safe.utils.OwnerKeyHandler
 import io.gnosis.safe.utils.ParamSerializer
 import javax.inject.Singleton
 
 @Singleton
 @Component(
-    modules = [
-        ApplicationModule::class,
-        InterceptorsModule::class,
-        ViewModelFactoryModule::class,
-        DatabaseModule::class,
-        RepositoryModule::class
-    ]
+        modules = [
+            ApplicationModule::class,
```

Comment:
what's with the formatting? Are you using https://github.com/gnosis/safe-android/blob/master/heimdall-style.xml?

Reply:
I thought, I was. Apparently I was not :-(